pkg/users/account-create-dialog.js

```diff
@@ -118,6 +130,23 @@
         return _("The full name must not contain colons.");
 }
 
+function validate_uid(uid, accounts, change) {
+    if (!uid)
+        return _("No UID specified");
+
+    const uid_number = Number(uid);
+    if (!Number.isInteger(uid_number) || uid_number < 0)
+        return _("UID must be a positive integer.");
+
+    if (uid_number < 1000)
+        return _("UID below 1000 is reserved for system users.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230215-154634-5d29fa30-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)